src/Pages/Settings/Addons.elm

```diff
@@ -28,13 +38,439 @@ type alias Params =
     ()
 
 
+save : Model -> Shared.Model -> Shared.Model
+save model shared =
+    shared
+
+
+load : Shared.Model -> Model -> ( Model, Cmd Msg )
+load shared model =
+    ( model, Cmd.none )
+
+
 type alias Model =
-    { route : Route }
+    { route : Route
+    , localSettingsList : LocalSettings
+    , aacToggle : Bool
+    , wmaToggle : Bool
+    , defaultToggle : Bool
+    , sensToggle : Bool
+    , inputToggle : Bool
+    , rtmpToggle : Bool
+    , universalAlbumToggle : Bool
+    , genericAlbumToggle : Bool
+    , localToggle : Bool
+    , universalArtistToggle : Bool
+    , genericArtistToggle : Bool
+    , musicToggle : Bool
+    , fanartToggle : Bool
+    , imdbToggle : Bool
+    , musicBrainzToggle : Bool
+    , audioToggle : Bool
+    , movieScraperToggle : Bool
+    , movieDatabaseToggle : Bool
+    , movieDatabsePythonToggle : Bool
+    , movieDatabseTVToggle : Bool
+    , tmdbToggle : Bool
+    , joystickToggle : Bool
+    , youtubeToggle : Bool
+    , addonToggle : Bool
+    , weatherToggle : Bool
+    , englishToggle : Bool
+    , uiToggle : Bool
+    , blackToggle : Bool
+    , dimToggle : Bool
+    , certifiToggle : Bool
+    , chardetToggle : Bool
+    , idnaToggle : Bool
+    , pythonImageToggle : Bool
+    , pythonCryptoToggle : Bool
+    , requestsToggle : Bool
+    , sixToggle : Bool
+    , urllibToggle : Bool
+    , versionToggle : Bool
+    , estouchyToggle : Bool
+    , estuaryToggle : Bool
+    , chorusToggle : Bool
+    }
+
+
+init : Shared.Model -> Url Params -> ( Model, Cmd Msg )
+init shared url =
+    ( { route = url.route
+      , localSettingsList = shared.addonLocalSettings
+      , aacToggle =
+            if getVal 1 shared.addonLocalSettings == "" then
```

Comment:
As all these seem to be completly copy and pasted, it would make sense to move the logic to a function and just pass the number into it